### PR TITLE
[BG] Started translation to Bulgarian

### PR DESCRIPTION
**Pull request by merkata**
Hi,

I've started translating the lessons into Bulgarian, I'll keep translating these days (I've set aside time for 3 docs per day) - let me know if it's useful and can be merged. Great site by the way!

Regards,

Mariyan

### CODE REVIEW

File: bg/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Concurrency
+category: advanced
+order: 4
+lang: en
```
Comment (doomspork) [inline bg/lessons/advanced/concurrency.md]: We unfortunately can't have these `lang: en` pages in here, they create issues with the generated site.

**Comment by doomspork** [comment]
Thank you @merkata!  I'll look this over today and give you some feedback, there's a few changes needed.

**Comment by nscyclone** [comment]
We have implemented a new feature that automatically generates the table of contents: #356.
This may help you avoid copypaste, typos and broken anchors. In order to use it you can replace the manually written TOC with `{% include toc.html %}`.

**Comment by doomspork** [comment]
Thank you @nscyclone

**Comment by doomspork** [comment]
Hi @merkata, would you like me to setup a GitHub Issue to help track translation progress like we've done with some other languages?  Here's the Russian example: https://github.com/doomspork/elixir-school/issues/220

Let me know when you're ready for this to be merged 👍

**Comment by merkata** [comment]
Hi @doomspork ,

That would be great, yes!

I've also made some changes to the bg/ files
* lang directive is now bg to avoid conflicts
* All documents in basic/ are using the automatic TOC
* I've removed the advanced/ and specific/ documents and I'll add the translated documents there gradually

Regards!

**Comment by doomspork** [comment]
@merkata sorry for the delay, I've been busy moving the last few weeks.  Is this good to be merged?

**Comment by merkata** [comment]
Hey @doomspork , absolutely, I think it's good to be merged. I also had some other activities, but now I'm going back to translating some more :)

**Comment by doomspork** [comment]
No worries @merkata, I've been quite being myself with my recent move.  I was just checking in to make sure I wasn't holding things up 😀

**Comment by doomspork** [comment]
Thank you again @merkata, sorry for the delay merging (life is busy!).

Do you have a twitter handle?  I'd like to tweet about the start of Bulgarian 😀

**Comment by merkata** [comment]
Hi @doomspork - handle is '@merkata', same as here :)

**Comment by doomspork** [comment]
Thanks @merkata!

**Comment by doomspork** [comment]
@merkata I just noticed that we didn't translate the config file so the section headers are absent on the website.  Could you open a new PR to fix that?

Starting here and going down: https://github.com/doomspork/elixir-school/blob/master/_config.yml#L11

**Pull request merged**